app/styles/main.scss

```diff
@@ -333,13 +333,33 @@ input.editableBox {
 .tumorButtonsGroup button span {
   font-weight: bold;
 }
-.geneComments .title{
-  font-weight: 400;
-  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
-}
-.geneComments .content{
-  font-weight: 600;
-  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+.geneComments {
+  .title{
+    font-weight: 400;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    width: 45%;
+    float: left;
+    .title-content {
+      width: 100%;
+      float: left;
+    }
+  }
+  .body{
+    width: 100%;
+    float: left;
+    margin: 5px 0;
+  }
+  .content{
+    font-weight: 600;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
I know this is from before. But the font-family should not be specified except it's expected

Reply:
I'll test on it.